coralproject/talk
# [CORL-2416]: Role promotions/demotions for site mods should have checkboxes

## What does this PR do?

These changes add checkboxes to the role change promotion/demotion modal for site moderators on the Community tab in the admin. This allows site moderators to have more fine-grained control over granting/removing moderation privileges for their site(s). This is for issue https://vmproduct.atlassian.net/browse/CORL-2416.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

As a site moderator (both single- and multi-site), promote and demote users on the Community tab in the admin. See that you can check/uncheck to add/remove sites to users' moderation scopes. Test out adding/removing privileges to users that have site moderator privileges outside of the scope of the viewer that you are signed in as.

See that admin user role change experience is still the same, and that it isn't affected by these changes.
 
## How do we deploy this PR?
